app/src/main/java/org/simple/clinic/recentpatientsview/RecentPatientsViewController.kt

```diff
@@ -89,16 +91,23 @@ class RecentPatientsViewController @Inject constructor(
         recentPatients
       }
 
-  private fun recentPatientItem(recentPatient: RecentPatient) =
-      RecentPatientItem(
-          uuid = recentPatient.uuid,
-          name = recentPatient.fullName,
-          age = age(recentPatient),
-          gender = recentPatient.gender,
-          updatedAt = recentPatient.updatedAt,
-          dateFormatter = dateFormatter,
-          clock = userClock
-      )
+  private fun recentPatientItem(recentPatient: RecentPatient): RecentPatientItem {
+    val today = Instant.now(userClock).toLocalDateAtZone(userClock.zone)
+    val patientCreatedAt = recentPatient.patientCreatedAt.toLocalDateAtZone(userClock.zone)
+
+    val isNewRegistration = today.isEqual(patientCreatedAt)
```

Comment:
We could also use `today == patientCreatedAt` to be more idiomatic.

Reply:
Done